.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 *.iml
+*.swp
```

Comment:
I don't use it personally, but I checked a couple other `.gitignore` files in hapi repos and didn't see it.